apis/apps/v1alpha1/cluster_types.go

```diff
@@ -600,6 +608,34 @@ type ClusterNetwork struct {
 	PubliclyAccessible bool `json:"publiclyAccessible,omitempty"`
 }
 
+type ServiceRef struct {
+	// name of the service reference declaration. references the serviceRefDeclaration name defined in clusterDefinition.componentDefs[*].serviceRefDeclarations[*].name
+	// +kubebuilder:validation:Required
+	Name string `json:"name"`
+
+	// namespace defines the namespace of the referenced Cluster or the namespace of the referenced ServiceDescriptor object.
+	// If not set, the referenced Cluster and ServiceDescriptor will be searched in the namespace of the current cluster by default.
+	// +optional
+	Namespace string `json:"namespace,omitempty" protobuf:"bytes,3,opt,name=namespace"`
+
+	// When referencing a service provided by other KubeBlocks cluster, you need to provide the name of the Cluster being referenced.
+	// By default, when other KubeBlocks Cluster are referenced, the ClusterDefinition.spec.connectionCredential secret corresponding to the referenced Cluster will be used to bind to the current component.
+	// currently, the connection credential secret needs to adhere to the following specification: The data field must contain the information for endpoint, port, username, and password.
```

Comment:
These should be optional, but not required.

Reply:
> These should be optional, but not required.

done